dll/win32/syssetup/wizard.c

```diff
@@ -2992,6 +2992,22 @@ ProcessSetupInf(
                              (LPBYTE)pSetupData->SourcePath,
                              (wcslen(pSetupData->SourcePath) + 1) * sizeof(WCHAR));
 
+        WCHAR szData[MAX_PATH];
+        LPWSTR pData = szData;
```

Comment:
Please move those two variable declarations at the top of the `if (res == ERROR_SUCCESS) { ` block.